docs/docs/default-actions.mdx

```diff
@@ -2,47 +2,60 @@
 id: default-actions
 sidebar_label: Default Actions
 title: Default Actions
+abstract: Default actions are actions that are built into the dialogue manager by default. Most of these are automatically predicted based on certain conversation situations. You may want to customise these to personalise your assitant.
```

Comment:
Check typo ci for these, as its catching the same british english and `assitant` as above